### ISSUE TITLE
Trying out the client.rs example of quiche and looks like I also having similar issue with establishing the connection. Client side output is:

### ISSUE THREAD

**Issue by saant03** [cloudflare/quiche#230]
Trying out the client.rs example of quiche and looks like I also having similar issue with establishing the connection. Client side output is: 

Trying to connecting to: https://quic.tech:4433/

>`2019-10-20T22:59:13.501943469Z TRACE quiche::tls] e8010d018eb490b7c1ea18c41d5829792e4663c2 tls write message lvl=Initial len=512
[2019-10-20T22:59:13.502030462Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 tx pkt Initial version=ff000017 dcid=ee570b6de2211d285133d055b7d36d69 scid=e8010d018eb490b7c1ea18c41d5829792e4663c2 len=1153 pn=0

> [2019-10-20T22:59:13.502115872Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 tx frm CRYPTO off=0 len=512

> [2019-10-20T22:59:13.502153552Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 tx frm PADDING len=621

> [2019-10-20T22:59:13.502266343Z TRACE quiche::recovery] e8010d018eb490b7c1ea18c41d5829792e4663c2 timer=999.462644ms crypto=1200 inflight=1200 cwnd=14520 latest_rtt=0ns srtt=None min_rtt=0ns rttvar=0ns probes=0 

>  -> initial handshake sent: 1200 bytes

> [2019-10-20T22:59:13.602666915Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 rx pkt Initial version=ff000017 dcid=e8010d018eb490b7c1ea18c41d5829792e4663c2 scid=3958fa21903a32d6c2fa86dd6fbd2e593b94a569 token= len=149 pn=0

> [2019-10-20T22:59:13.603008113Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 rx frm ACK delay=311 blocks=[0..0]

> [2019-10-20T22:59:13.603175747Z TRACE quiche::recovery] e8010d018eb490b7c1ea18c41d5829792e4663c2 packet newly acked 0

> [2019-10-20T22:59:13.603277447Z TRACE quiche::recovery] e8010d018eb490b7c1ea18c41d5829792e4663c2 timer=none crypto=0 inflight=0 cwnd=15720 latest_rtt=100.850333ms srtt=Some(100.850333ms) min_rtt=100.850333ms rttvar=50.425166ms probes=0 

> [2019-10-20T22:59:13.603505423Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 rx frm CRYPTO off=0 len=122

> [2019-10-20T22:59:13.604675322Z TRACE quiche::tls] e8010d018eb490b7c1ea18c41d5829792e4663c2 tls set encryption secret lvl=Handshake

> Quiche Connection stats are: recv=1 sent=1 lost=0 rtt=100.850333ms

> Connection established? -> false

> Trace_ID is: "e8010d018eb490b7c1ea18c41d5829792e4663c2"

> [2019-10-20T22:59:13.605002373Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 tx pkt Initial version=ff000017 dcid=3958fa21903a32d6c2fa86dd6fbd2e593b94a569 scid=e8010d018eb490b7c1ea18c41d5829792e4663c2 len=1149 pn=1

> [2019-10-20T22:59:13.605206375Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 tx frm ACK delay=297 blocks=[0..0]

> [2019-10-20T22:59:13.605313115Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 tx frm PADDING len=1127

> [2019-10-20T22:59:13.605646508Z TRACE quiche::recovery] e8010d018eb490b7c1ea18c41d5829792e4663c2 timer=none crypto=0 inflight=0 cwnd=15720 latest_rtt=100.850333ms srtt=Some(100.850333ms) min_rtt=100.850333ms rttvar=50.425166ms probes=0 

> Wrote 1200 bytes

> [2019-10-20T22:59:13.605881388Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 rx pkt Handshake version=ff000017 dcid=e8010d018eb490b7c1ea18c41d5829792e4663c2 scid=3958fa21903a32d6c2fa86dd6fbd2e593b94a569 len=1147 pn=0

> [2019-10-20T22:59:13.606122509Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 rx frm CRYPTO off=0 len=1126

> Quiche Connection stats are: recv=2 sent=2 lost=0 rtt=100.850333ms

> Connection established? -> false

> Trace_ID is: "e8010d018eb490b7c1ea18c41d5829792e4663c2"

> [2019-10-20T22:59:13.606323314Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 tx pkt Handshake version=ff000017 dcid=3958fa21903a32d6c2fa86dd6fbd2e593b94a569 scid=e8010d018eb490b7c1ea18c41d5829792e4663c2 len=21 pn=0

> [2019-10-20T22:59:13.606523008Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 tx frm ACK delay=55 blocks=[0..0]

> [2019-10-20T22:59:13.606660555Z TRACE quiche::recovery] e8010d018eb490b7c1ea18c41d5829792e4663c2 timer=none crypto=0 inflight=0 cwnd=15720 latest_rtt=100.850333ms srtt=Some(100.850333ms) min_rtt=100.850333ms rttvar=50.425166ms probes=0 

> [2019-10-20T22:59:13.606862263Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 dropped epoch 0 state

> Wrote 70 bytes

> [2019-10-20T22:59:13.606998046Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 rx pkt Handshake version=ff000017 dcid=e8010d018eb490b7c1ea18c41d5829792e4663c2 scid=3958fa21903a32d6c2fa86dd6fbd2e593b94a569 len=1148 pn=1

> [2019-10-20T22:59:13.607236673Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 rx frm CRYPTO off=1126 len=1126

> Quiche Connection stats are: recv=3 sent=3 lost=0 rtt=100.850333ms
> Connection established? -> false
> Trace_ID is: "e8010d018eb490b7c1ea18c41d5829792e4663c2"

> [2019-10-20T22:59:13.607428171Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 tx pkt Handshake version=ff000017 dcid=3958fa21903a32d6c2fa86dd6fbd2e593b94a569 scid=e8010d018eb490b7c1ea18c41d5829792e4663c2 len=21 pn=1

> [2019-10-20T22:59:13.607627033Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 tx frm ACK delay=53 blocks=[0..1]

> [2019-10-20T22:59:13.607761445Z TRACE quiche::recovery] e8010d018eb490b7c1ea18c41d5829792e4663c2 timer=none crypto=0 inflight=0 cwnd=15720 latest_rtt=100.850333ms srtt=Some(100.850333ms) min_rtt=100.850333ms rttvar=50.425166ms probes=0 

> Wrote 70 bytes

> [2019-10-20T22:59:13.607981466Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 rx pkt Handshake version=ff000017 dcid=e8010d018eb490b7c1ea18c41d5829792e4663c2 scid=3958fa21903a32d6c2fa86dd6fbd2e593b94a569 len=701 pn=2

> [2019-10-20T22:59:13.608226795Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 rx frm CRYPTO off=2252 len=679

> [2019-10-20T22:59:13.609306384Z TRACE quiche::tls] e8010d018eb490b7c1ea18c41d5829792e4663c2 tls set encryption secret lvl=OneRTT

> [2019-10-20T22:59:13.609560790Z TRACE quiche::tls] e8010d018eb490b7c1ea18c41d5829792e4663c2 tls write message lvl=Handshake len=36

> [2019-10-20T22:59:18.609123713Z TRACE quiche] e8010d018eb490b7c1ea18c41d5829792e4663c2 idle timeout expired`

_Originally posted by @saant03 in https://github.com/cloudflare/quiche/issues/95#issuecomment-544304423_

**Comment by junhochoi** [comment]
Can you post a full command line? It looks working for me:
```
$ RUST_LOG=info target/debug/examples/client https://quic.tech:4433
[2019-10-22T16:33:17.700703000Z INFO  client] connecting to [2001:19f0:5001:925:5400:1ff:fea6:5b54]:4433 from [2601:647:4900:1f01:7d44:3e7c:439b:df8]:61080 with scid ef2f3804ee3c3093fe1ddfac2f4c0d29e714a01f
[2019-10-22T16:33:18.027921000Z INFO  client] sending HTTP request for /
Not Found!
[2019-10-22T16:33:18.189625000Z INFO  client] response received in 488.412335ms, closing...
[2019-10-22T16:33:19.296092000Z INFO  client] connection closed, recv=6 sent=6 lost=0 rtt=160.942809ms
```

**Comment by ghedo** [comment]
Closing as there was no reply for over a month. We can reopen later if the reporter comes back.
